server/encryptionkm/key_manager.go

```diff
@@ -488,6 +503,7 @@ func (m *KeyManager) SetLeadership(leadership *election.Leadership) error {
 type keyManagerHelper struct {
 	now                          func() time.Time
 	tick                         func(ticker *time.Ticker) <-chan time.Time
+	newMasterKey                 func(*encryptionpb.MasterKey, []byte) (*encryption.MasterKey, error)
```

Comment:
Why do we need to add this to the helper?

Reply:
It is used in the unit test to fake and check ciphertext_key.